app/models/school_data_importer.rb

```diff
@@ -45,6 +45,7 @@ def row_to_school(row)
     school.local_authority_district = local_authority_district
     school.close_date = row.fetch("CloseDate")
     school.establishment_number = row.fetch("EstablishmentNumber")
+    school.statutory_high_age = row.fetch("StatutoryHighAge")
```

Comment:
Once this code is deployed do we want to trigger a school import to get these values set or will it be OK to wait for the nightly import?

Reply:
Now we have access I think it would be good to run the import. We can then let the claimants who are waiting on this to be resolved know they can claim.

Reply:
Sounds good 👍🏽